projects/uitdatabank/reference/entry.json

```diff
@@ -9496,6 +9496,320 @@
+                                "label": "Elsene + deelgemeenten",
+                                "value": "nis-21009"
+                              },
+                              {
+                                "label": "Etterbeek + deelgemeenten",
+                                "value": "nis-21005"
+                              },
+                              {
+                                "label": "Evere + deelgemeenten",
+                                "value": "nis-21006"
+                              },
+                              {
+                                "label": "Ganshoren + deelgemeenten",
+                                "value": "nis-21008"
+                              },
+                              {
+                                "label": "Jette + deelgemeenten",
+                                "value": "nis-21010"
+                              },
+                              {
+                                "label": "Koekelberg + deelgemeenten",
+                                "value": "nis-21011"
+                              },
+                              {
+                                "label": "Oudergem + deelgemeenten",
+                                "value": "nis-21002"
+                              },
+                              {
+                                "label": "Schaarbeek + deelgemeenten",
+                                "value": "nis-21015"
+                              },
+                              {
+                                "label": "Sint-Agatha-Berchem + deelgemeenten",
+                                "value": "nis-21003"
+                              },
+                              {
+                                "label": "Sint-Gillis + deelgemeenten",
+                                "value": "nis-21013"
+                              },
+                              {
+                                "label": "Sint-Jans-Molenbeek + deelgemeenten",
+                                "value": "nis-21012"
+                              },
+                              {
+                                "label": "Sint-Joost-ten-Node + deelgemeenten",
+                                "value": "nis-21014"
+                              }
+                            ]
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                }
+              }
+            }
+          }
+        },
+        "operationId": "get-regions",
+        "x-internal": true,
+        "parameters": [],
+        "description": "Returns a hierarchical list of the regions in Flanders and Brussel. Each level includes a label, value with the NIS-code, and optional nested children, allowing integrators to display lists of all regions and municipalities."
+      }
+    },
+    "/cultuurkuur/education-levels": {
+      "get": {
+        "summary": "/cultuurkuur/education-levels",
```

Comment:
Same remark here, something like `educationLevels - GET`, but the URL path `education-levels`